docker/py3-native/verify.py

```diff
@@ -0,0 +1,165 @@
+# chromium
+from selenium import webdriver
+from selenium.common.exceptions import NoSuchElementException, InvalidArgumentException
+from PyPDF2 import PdfFileReader
+from pdf2image import convert_from_path
+import numpy as np
+from PIL import Image
+import sys
+import base64
+import subprocess
+
+# verify the google-chrome and chromedriver have the same version (excluding patch level)
+chrome_version = subprocess.check_output(["google-chrome", "--version"], text=True).split()[2]
+driver_version = subprocess.check_output(["/usr/bin/chromedriver", "--version"], text=True).split()[1]
+
+print(f'Comparing full versions: {chrome_version} to: {driver_version}')
+chrome_version_arr = chrome_version.split('.')[:3]
+driver_version_arr = driver_version.split('.')[:3]
+print(f'Comparing versions without patch: {chrome_version_arr} to: {driver_version_arr}')
+assert chrome_version_arr == driver_version_arr
+print('chromium is good!!!')
+
+
+# tesseract
+subprocess.check_output(["tesseract", "--version", "--list-langs"], text=True)
+
+# crypto
+from cryptography.fernet import Fernet
+import msal
+from bs4 import BeautifulSoup
+# Make sure cryptograph works
+key = Fernet.generate_key()
+print("All is good. cryptography generated a key: {}".format(key))
+print(msal.oauth2cli.assertion.JwtAssertionCreator('', 'All is good. msal was imported successfully').algorithm)
+print('crypto is good!!!')
+
+
+# readpdf
+from pikepdf import Pdf
+from PyPDF2 import PdfFileReader, PdfFileWriter
+print('readpdf is good!!!')
+
+
+# parse-emails
+from parse_emails.parse_emails import EmailParser
+print('parse-emails is good!!!')
+
+
+# docxpy + office-utils
+from docx import Document
+# make sure libreoffice is installed
+subprocess.check_output(["soffice", "--version"], text=True)
+print('docxpy + office-utils are good!!!')
+
+
+# sklearn
+import sklearn
+import pandas
+import numpy
+import nltk
+import dill
+import eli5
+import networkx
+print('sklearn is good!!!')
```

Comment:
can we import these tests from where they sit in the repo instead of copying them to here?

Reply:
That is a bit problematic, what if someone adds a new dependency to image X, but in the native image we still do not support it? the `verify.py` will fail. If you think that this is good (and then we will know that this dependency needs to be added) i can do it with imports

Reply:
also, i will need to copy the `verify.py` files to the container, or make it with a symbolic link, however symbolic link would work only a single file and not on all of them. What you want to achieve here is a bit complicated.

Reply:
In the case you described, this either will be a reminder to update the master image as well, or we should drop support for that image if the tests dont pass and we dont want to support it
Cant we copy the files in in the same place we do verify.py?
@tkatzir  what do you think?

If this is too high effort for now lets open a task to do it after

Reply:
we will need to touch build code so the docker has the files

Reply:
I think we can do it with symlinks - testing it now. see the latest commit, if that works we can do a symbolic link to each docker image we would like.

Reply:
looks like it works, i updated the docker build a bit to run multiple `verify.py` files